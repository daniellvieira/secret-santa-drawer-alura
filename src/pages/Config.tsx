import Card from "../components/Card";
import Footer from "../components/Footer";
import Form from "../components/Form";
import UsersList from "../components/UsersList";

const Config = () => {
  return (
    <Card>
      <section>
        <h2>Vamos come√ßar!</h2>
        <Form />
        <UsersList />
        <Footer />
      </section>
    </Card>
  )
}

export default Config;