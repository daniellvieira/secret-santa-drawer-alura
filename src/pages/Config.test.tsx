import { render } from "@testing-library/react";
import { RecoilRoot } from "recoil"
import Config from "./Config";

const mockNavigation = jest.fn();
jest.mock('react-router-dom', () => {
  return { useNavigate: () => mockNavigation }
})

describe('the page of configurations', () => {
  test('should be render correctly', () => {
    const { container } = render(<RecoilRoot><Config /></RecoilRoot>)

    expect(container).toMatchSnapshot();
  })
})