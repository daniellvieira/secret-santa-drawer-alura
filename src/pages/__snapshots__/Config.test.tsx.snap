// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the page of configurations should be render correctly 1`] = `
<div>
  <div
    class="card"
  >
    <section>
      <h2>
        Vamos come√ßar!
      </h2>
      <form>
        <div
          class="group-input-btn"
        >
          <input
            placeholder="Insira os nomes dos participantes"
            type="text"
            value=""
          />
          <button
            disabled=""
          >
            Adicionar
          </button>
        </div>
        
      </form>
      <ul />
      <footer
        class="footer-configs"
      >
        <button
          class="btn"
          disabled=""
        >
          Iniciar brincadeira!
        </button>
        <img
          alt="Sacolas de compras"
          src="/images/sacolas.png"
        />
      </footer>
    </section>
  </div>
</div>
`;
